Example/Sources/ConversationViewController.swift

```diff
@@ -440,12 +440,25 @@ extension ConversationViewController: MessageLabelDelegate {
 extension ConversationViewController: MessageInputBarDelegate {
 
     func messageInputBar(_ inputBar: MessageInputBar, didPressSendButtonWith text: String) {
-        let attributedText = NSAttributedString(string: text, attributes: [.font: UIFont.systemFont(ofSize: 8), .foregroundColor: UIColor.blue])
-        let id = UUID().uuidString
-        let message = MockMessage(attributedText: attributedText, sender: currentSender(), messageId: id, date: Date())
-        messageList.append(message)
+        
+        // Each NSTextAttachment that contains an image will count as one empty character in the text: String
+        if text.count > inputBar.inputTextView.images.count {
+            
+            let attributedText = NSAttributedString(string: text, attributes: [.font: UIFont.systemFont(ofSize: 8), .foregroundColor: UIColor.blue])
+            
+            let message = MockMessage(text: text, sender: currentSender(), messageId: UUID().uuidString, date: Date())
```

Comment:
Should use `attributeText` instead of `text` when create `MockMessage` instance.